components/Slider/Slider.js

```diff
@@ -42,6 +36,12 @@ export default class Slider {
           </div>
           : null
         }
+        <ReactSlider withBars
```

Comment:
It's only alpha as it's been called that by the maintainer. Works fine as far as I can tell?